tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/jms/server/config/JMSServerPropertyConfigTest.java

```diff
@@ -78,7 +78,31 @@ public void testConfigViaBrokerPropertiesFromClasspath() throws Exception {
          server.setConfiguration(configuration);
          server.start();
 
-         assertEquals("ConfiguredViaProperties", server.getActiveMQServer().getConfiguration().getName());
+         assertEquals(-2, server.getActiveMQServer().getConfiguration().getGracefulShutdownTimeout());
```

Comment:
Would be good to add a constant with/or comment to explain where the -2 comes from, will be very non-obvious later (isnt even that obvious now, until the diff following later to explain it in a totally different file).

Should probably also assert that ActiveMQDefaultConfiguration.getDefaultGracefulShutdownTimeout() isnt -2, otherwise this test and the next one could be checking the same thing and not really verifying that "server.setPropertiesResourcePath(null);" does anything.

(The tests are so similar they could also share a common impl and pass a boolean to say whether to set the value and govern the expected result..might make them more obvious).

Reply:
the hint is in the name, and most config defaults to -1, -2 is plainly odd just for that reason!

Reply:
Why hint when you can straight up say so. Here where it is being asserted upon, rather than burried away in some other file entirely that you would already need to know about to get there.